pages/test_analytics/javascript_collectors.md

```diff
@@ -55,9 +55,11 @@ To configure Jest:
 
     ```json
     {
-        "reporters": ["default", "buildkite-test-collector/jest/reporter"]
+        "reporters": ["default", "buildkite-test-collector/jest/reporter"],
+        "testLocationInResults": true,
     }
     ```
+    **Note:** The `"testLocationInResults": true` setting enables column and line capture for Test Analytics.
```

Comment:
Hopefully this note is OK?

Anyway, I'll approve this PR so that you can merge it in.